lib/TopStoriesFeed.jsm

```diff
@@ -112,6 +112,13 @@ this.TopStoriesFeed = class TopStoriesFeed {
     this.store.dispatch(shouldBroadcast ? ac.BroadcastToContent(action) : ac.AlsoToPreloaded(action));
   }
 
+  dispatchLayoutUpdate(data, shouldBroadcast) {
```

Comment:
For improved readability of calling code, this I'd suggest calling this `maybeDispatchLayoutUpdate`, since it may or may not call dispatch under the hood.